backend/api/groups.py

```diff
@@ -0,0 +1,29 @@
+# backend/api/groups.py
+from fastapi import APIRouter, HTTPException
+from starlette import status
+from bson import ObjectId
+
+from backend.api.api_request_classes import CreateGroupRequest
+from backend.common import config_info
+from backend.common.models import Group
+
+router = APIRouter(prefix="/api/v1/groups", tags=["groups"])
+db = config_info.get_db()
+
+
+@router.post("/create_group", status_code=status.HTTP_201_CREATED)
+async def create_group(request: CreateGroupRequest):
```

Comment:
This request should be of type Group, CreateGroupRequest is not used as expected